### PR TITLE
feat the grid improvements 3

### PR DESCRIPTION
**Pull request by nastita**
### Description

- replaced event listener for resize observer
- Simplified edit controls, only allow drag from handle
- Refactored grid debug modal
- remove unused ref

### CODE REVIEW

File: components/GridView.vue [hunk 1]
```diff
@@ -167,19 +160,57 @@ onUnmounted(() => {
...
+        disabled="true"
+      >
+        <lukso-icon name="plus" size="medium" class="mx-1"></lukso-icon>
+      </lukso-button>
       <lukso-button
         size="small"
         type="button"
         variant="secondary"
+        is-icon
+        @click="toggleEditMode()"
```
Comment (dzbo) [inline components/GridView.vue:198]:
```suggestion
        @click="toggleEditMode"
```

We prefix click functions with `handle`. Brackets are not necessary.

File: components/GridView.vue [hunk 2]
```diff
@@ -131,29 +122,31 @@ function clearSelection(): void {
   window.getSelection()?.removeAllRanges()
 }
 
-onMounted(() => {
-  window.addEventListener('resize', resizeHandler)
-})
+function toggleEditMode(): void {
```
Comment (dzbo) [inline components/GridView.vue:125]: Please change to arrow function. Also the one above.

**Review by dzbo — CHANGES_REQUESTED** [review]

File: components/GridView.vue (hunk 2 above)
Comment (nastita) [reply components/GridView.vue:125]: all "handlers" should be defined as arrow functions?
asking for future prs

Comment (nastita) [reply components/GridView.vue]: updated

Comment (dzbo) [reply components/GridView.vue]: > all "handlers" should be defined as arrow functions? asking for future prs

yes please. We want to keep consistent code style across components.

**Review by dzbo — APPROVED** [review]

**Pull request merged**